tests/test_decorator.py

```diff
@@ -23,18 +24,18 @@ def test_datetime() -> None:
         assert response.headers.get("X-FastAPI-Cache") == "MISS"
         now = response.json().get("now")
         now_ = pendulum.now()
-        assert pendulum.parse(now) == now_
+        assert pendulum.parse(now).to_atom_string() == now_.to_atom_string()
```

Comment:
Why do we need to convert `to_atom_string` since we are comparing datetime values?